src/probnum/problems/_problems.py

```diff
@@ -1 +1,157 @@
-"""Definitions of problems solved by probabilistic numerical methods."""
+"""Definitions of problems currently solved by probabilistic numerical methods.
+
+If a problem is not listed here, we have not implemented a probabilistic
+numerical method for it yet.
+"""
+
+import dataclasses
```

Comment:
the import is the same for all python versions (even though for 3.6 we need the additional dependency `import dataclasses` -- see `setup.cfg`).